src/WatcherCommand.php

```diff
@@ -8,9 +8,12 @@
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
+use function var_dump;
 
 class WatcherCommand extends Command
 {
+    const PHPUNIT_WATCH_CONFIG_FILENAME = "/.phpunit-watcher.yml";
```

Comment:
The class is kinda small, this constant is only being used in one method. That's why I think that using a constant here is a bit overkill. Use a regular variable to hold this value in the `getOptionsConfigFile` method itself.